### DISCUSSION TITLE
Tap to complete task in View-Mode / Edit check in ViewMode

### DISCUSSION THREAD

**Discussion by patrick-haldi** [gsantner/markor#1644] [Q&A] [answered]
In a normal markdown file, it would be awesome if you could just tap the box in view mode to mark the task as done. Would be much more intuitive than switching to edit mode and adding an x (or tapping the task button)

**Comment by gsantner** [comment, marked as answer]
View mode is view mode and it will not get edit functionality. this has been asked numerous times already

**Reply by patrick-haldi** [reply]
Maybe the fact that it has been asked numerous times could be an indicator that it's a user need/expectation. Just thinking from a UX viewpoint..

**Reply by gsantner** [reply]
People ask for a lot and cry a lot. That doesn't mean everything has todo everything people wish for.

If I would do everything like wished, still not everybody would be happy. And Markor would be not much different to other applications, bloated as hell, stuffed with lots of dependencies, loaded with every providers own brew out there (dropbox, ms, gdrive, .....).  

People have lots of expectations and it's impossible to make everyone happy. Adding one thing, kills other thing.  
Accept it. Deal with it. You dont get everything you want, even if **you** think its the better.

If you want something that does something differently, use something differently. This is not Microsoft Word and it will never be. And neither it will be one of those countless generic todo apps.

The principle and what is granted is very simple. View mode is for viewing, and is read-only. I don't want to start making exceptions to this. No file write access from web to files.

**Reply by patrick-haldi** [reply]
I agree with everything you say, and it's the main reason I like your app. 
It's just.. Checkboxes that can't be checked without switching to edit mode and back.. Bit weird. But you do you

**Reply by teggyv** [reply]
I understand the point about no write access from web to files; nevertheless, in case this can be on any weight, I'll second the point about having checkboxes available in view mode (couple of apps with similar dev background, qownnotes is one of them, provide this handy tweak). That said, keep up the good work; the app rocks !

**Comment by harshad1** [comment]
Note that you can use a html checkbox to get this effect

```
# Bike ride checklist

## Basics
- <input type="checkbox"> Keys
- <input type="checkbox"> Pannier
- <input type="checkbox"> Helmet
- <input type="checkbox"> Glasses
- <input type="checkbox"> Water
```

The resulting checkboxes are checkable in view mode (but will revert if you go to edit mode and back).

This is extremely useful for checklists.

**Comment by gsantner** [comment]
No. It has been asked numerous times, please search for more information.

**Comment by harshad1** [comment]
@gsantner Would it be possible to jump from view mode to the corresponding task in edit mode? i.e. click / long click on 4th task in view mode jumps to 4th task in edit mode.

If this is feasible it may partially resolve this need

**Reply by gsantner** [reply]
It's not feasible, Markor is not a HTML-to-?/Anything converter. HTML/output not directly correlates to the markup input. I outlined it in some discussion(s) pretty detailed already, not sure if it's this one.
